opentracing-jaxrs2-itest/auto-discovery/src/test/java/io/opentracing/contrib/jaxrs/itest/auto/discovery/SwarmAutoDiscoveryITest.java

```diff
@@ -29,6 +33,8 @@ public void testRequestIsTraced() throws InterruptedException, NamingException {
 
         client.close();
 
-        Assert.assertEquals(1, mockTracer.finishedSpans().size());
+        List<MockSpan> spans = mockTracer.finishedSpans();
+        Assert.assertEquals(3, spans.size());
```

Comment:
Before there was only one span? Why 3 now?

Reply:
Honestly I'm not sure. Reading the code, I'm not sure why it was ever 1 before. Those extra spans come from the tracing in `TracingInterceptor`, and its involvement hasn't changed.

Reply:
I am not sure why this was passing. Serialization tracing is enabled by default so you did it right.